extensions/wikia/GlobalNavigation/templates/GlobalNavigation_hubsMenuSections.php

```diff
@@ -3,16 +3,16 @@
 		<div class="hub-menu-columns">
 			<? if ( array_key_exists( 'children', $sections ) ) : ?>
 				<? foreach( $sections['children'] as $category ): ?>
-					<strong><?= $category['text'] ?></strong>
-					<? if ( array_key_exists( 'children', $category ) ) : ?>
-						<ul>
+					<ul>
+						<li><strong><?= $category['text'] ?></strong></li>
```

Comment:
It was like this before, I'm just partially reverting change that was implemented. I don't want to complicate things at this point. It would probably make more sense to replace UL with DIV or SECTION tags (as this is one section inside menu), but it's too complicated for quick fix.